Stock-signalling.py

```diff
@@ -0,0 +1,57 @@
+import requests
+
+
+# List ofstock symbols that will be monitored
+stock_symbols = ["TSLA", "AAPL", "MSFT", "GOOGL", "NKE"]
+
+# API key and the base URL
+alpha_vantage_api_key = "NZXLRSMAPCX4T3YW"

```

Comment:
Nice use of variables, so you don't have to write the API Key every time you want to call it. 
I just learned about environment variables (.env files), they are good for storing sensitive information in a secure way.
I suggest storing your API key in an environment variable for added security.